src/const/testing.js

```diff
@@ -1,4 +1,4 @@
 // FIXME: temporary variables for hiding functional
 export const HIDE_EXPORT_BUTTON = true;
 export const HIDE_FILTERS = true;
-export const HIDE_SELECT_COLUMNS = true;
+export const HIDE_SELECT_COLUMNS = false;
```

Comment:
here

Reply:
planned to restore that before merging). Restored